src/meta/src/barrier/command.rs

```diff
@@ -97,6 +97,8 @@ pub struct ReplaceTablePlan {
     /// Note that there's no `SourceBackfillExecutor` involved for table with connector, so we don't need to worry about
     /// `backfill_splits`.
     pub init_split_assignment: SplitAssignment,
+    pub streaming_job: StreamingJob,
+    pub dummy_id: u32,
```

Comment:
Curious why it's named `dummy_id`? What does it mean?

Reply:
And any reason to add these information in the `ReplaceTablePlan` command? I think in all commands that need to track the status of backfill, there should be a `CreateStreamingJob` command, and we can get the `StreamingJob` information there. What ddl would only have  the `ReplaceTablePlan` command and we have to store the `StreamingJob` here in the `ReplaceTablePlan` command?